website/dynamic-blocks/headings.js

```diff
@@ -4,44 +4,40 @@ import { Fragment } from 'react';
 import { jsx } from '@westpac/core';
 import { Heading as WestpacHeading } from '@westpac/heading';
 import { Cell, Grid } from '@westpac/grid';
+import { Body } from '@westpac/body';
 import Select from '@arch-ui/select';
 import { FieldContainer, FieldLabel, FieldInput } from '@arch-ui/fields';
 import { CheckboxPrimitive } from '@arch-ui/controls';
 import { Input } from '@arch-ui/input';
 
-const Component = ({ heading, size, level, addTableContent, subText, text }) => {
+const Component = ({ heading, size, level, addTableContent, indent = true, subText, text }) => {
```

Comment:
@MadeByMike I've modified this block to take a new value, just wanted to make sure adding the default value will make sure things don't break

Reply:
Yep, you've added a default! You're good :)